diff --git plugins/opCommunityTopicPlugin/apps/pc_frontend/modules/communityEventComment/templates/_list.php plugins/opCommunityTopicPlugin/apps/pc_frontend/modules/communityEventComment/templates/_list.php
index 16e39af..2c4e439 100644
--- plugins/opCommunityTopicPlugin/apps/pc_frontend/modules/communityEventComment/templates/_list.php
+++ plugins/opCommunityTopicPlugin/apps/pc_frontend/modules/communityEventComment/templates/_list.php
@@ -39,6 +39,17 @@ $images = $comment->getImages();
 <?php echo op_url_cmd(nl2br($comment->getBody())) ?>
 </p>
 </div>
+<!--Like Plugin -->
+<div class="like" style="display: none;">
+<span class="like-wrapper" data-like-id="<?php echo $comment->getId() ?>" data-like-target="e" member-id="<?php echo $comment->member_id ?>">
+<span class="like-post">いいね！</span>
+<span class="like-cancel">いいね！を取り消す&nbsp;</span>
+<span class="like-you">あなたが「いいね！」と言っています。</span><br />
+<a class="like-list" href="javascript:void(0)" onclick="$('#likeModal').modal('show')"></a>
+<div class="like-list-member"></div>
+<span class="like-friend-list"></span>
+</span>
+</div>
 </dd>
 </dl>
 <?php endforeach; ?>
@@ -48,3 +59,23 @@ $images = $comment->getImages();
 </div>
 </div>
 <?php endif; ?>
+
+<script id="LikelistTemplate" type="text/x-jquery-tmpl">
+<table>
+<tr style="padding: 2px;">
+<td style="width: 48px; padding: 2px;"><a href="${profile_url}"><img src="${profile_image}" width="48"></a></td>
+<td style="padding: 2px;"><a href="${profile_url}">${name}</a></td>
+</tr>
+</table>
+</script>
+
+<div id="likeModal" class="modal hide">
+  <div class="modal-header">
+    <h1>「いいね！」と言っている人</h1>
+  </div>
+  <div class="like-modal-body">
+  </div>
+  <div class="modal-footer">
+    <a href="#" class="btn close" data-dismiss="modal" aria-hidden="true">閉じる</a>
+  </div>
+</div>
diff --git plugins/opCommunityTopicPlugin/apps/pc_frontend/modules/communityTopicComment/templates/_list.php plugins/opCommunityTopicPlugin/apps/pc_frontend/modules/communityTopicComment/templates/_list.php
index 1c3c31d..c6af9da 100644
--- plugins/opCommunityTopicPlugin/apps/pc_frontend/modules/communityTopicComment/templates/_list.php
+++ plugins/opCommunityTopicPlugin/apps/pc_frontend/modules/communityTopicComment/templates/_list.php
@@ -38,6 +38,17 @@ $images = $comment->getImages();
 <?php echo op_url_cmd(nl2br($comment->getBody())) ?>
 </p>
 </div>
+<!--Like Plugin -->
+<div class="like" style="display: none;">
+<span class="like-wrapper" data-like-id="<?php echo $comment->getId() ?>" data-like-target="t" member-id="<?php echo $comment->member_id ?>">
+<span class="like-post">いいね！</span>
+<span class="like-cancel">いいね！を取り消す&nbsp;</span>
+<span class="like-you">あなたが「いいね！」と言っています。</span><br />
+<a class="like-list" href="javascript:void(0)" onclick="$('#likeModal').modal('show')"></a>
+<div class="like-list-member"></div>
+<span class="like-friend-list"></span>
+</span>
+</div>
 </dd>
 </dl>
 <?php endforeach; ?>
@@ -47,3 +58,23 @@ $images = $comment->getImages();
 </div>
 </div>
 <?php endif; ?>
+
+<script id="LikelistTemplate" type="text/x-jquery-tmpl">
+<table>
+<tr style="padding: 2px;">
+<td style="width: 48px; padding: 2px;"><a href="${profile_url}"><img src="${profile_image}" width="48"></a></td>
+<td style="padding: 2px;"><a href="${profile_url}">${name}</a></td>
+</tr>
+</table>
+</script>
+
+<div id="likeModal" class="modal hide">
+  <div class="modal-header">
+    <h1>「いいね！」と言っている人</h1>
+  </div>
+  <div class="like-modal-body">
+  </div>
+  <div class="modal-footer">
+    <a href="#" class="btn close" data-dismiss="modal" aria-hidden="true">閉じる</a>
+  </div>
+</div>
