diff --git plugins/opDiaryPlugin/apps/pc_frontend/modules/diary/templates/showSuccess.php plugins/opDiaryPlugin/apps/pc_frontend/modules/diary/templates/showSuccess.php
index 258d189..941aad0 100644
--- plugins/opDiaryPlugin/apps/pc_frontend/modules/diary/templates/showSuccess.php
+++ plugins/opDiaryPlugin/apps/pc_frontend/modules/diary/templates/showSuccess.php
@@ -36,6 +36,17 @@
 <?php endif; ?>
 <?php echo op_url_cmd(op_decoration(nl2br($diary->body))) ?>
 </div>
+<!--Like Plugin -->
+<div class="like" style="display: none;">
+<span class="like-wrapper" data-like-id="<?php echo $diary->getId() ?>" data-like-target="D" member-id="<?php echo $diary->member_id ?>">
+<span class="like-post">いいね！</span>
+<span class="like-cancel">いいね！を取り消す&nbsp;</span>
+<span class="like-you">あなたが「いいね！」と言っています。</span><br />
+<a class="like-list" href="javascript:void(0)" onclick="$('#likeModal').modal('show')"></a>
+<div class="like-list-member"></div>
+<span class="like-friend-list"></span>
+</span>
+</div>
 </dd>
 </dl>
 <?php if ($diary->member_id === $myMemberId): ?>
@@ -68,3 +79,23 @@ op_include_form('formDiaryComment', $form, array(
 <?php endif; ?>
 
 <?php op_include_line('lineLinkToDiaryMemberList', link_to(__('Diaries of %1%', array('%1%' => $member->name)), 'diary_list_member', $member)) ?>
+
+<script id="LikelistTemplate" type="text/x-jquery-tmpl">
+<table>
+<tr style="padding: 2px;">
+<td style="width: 48px; padding: 2px;"><a href="${profile_url}"><img src="${profile_image}" width="48"></a></td>
+<td style="padding: 2px;"><a href="${profile_url}">${name}</a></td>
+</tr>
+</table>
+</script>
+
+<div id="likeModal" class="modal hide">
+  <div class="modal-header">
+    <h1>「いいね！」と言っている人</h1>
+  </div>
+  <div class="like-modal-body">
+  </div>
+  <div class="modal-footer">
+    <a href="#" class="btn close" data-dismiss="modal" aria-hidden="true">閉じる</a>
+  </div>
+</div>
diff --git plugins/opDiaryPlugin/apps/pc_frontend/modules/diaryComment/templates/_list.php plugins/opDiaryPlugin/apps/pc_frontend/modules/diaryComment/templates/_list.php
index dc648ca..6560e78 100644
--- plugins/opDiaryPlugin/apps/pc_frontend/modules/diaryComment/templates/_list.php
+++ plugins/opDiaryPlugin/apps/pc_frontend/modules/diaryComment/templates/_list.php
@@ -55,9 +55,40 @@
 <?php echo op_url_cmd(nl2br($comment->body)) ?>
 </p>
 </div>
+<!--Like Plugin -->
+<div class="like" style="display: none;">
+<span class="like-wrapper" data-like-id="<?php echo $comment->getId() ?>" data-like-target="d" member-id="<?php echo $comment->member_id ?>">
+<span class="like-post">いいね！</span>
+<span class="like-cancel">いいね！を取り消す&nbsp;</span>
+<span class="like-you">あなたが「いいね！」と言っています。</span><br />
+<a class="like-list" href="javascript:void(0)" onclick="$('#likeModal').modal('show')"></a>
+<div class="like-list-member"></div>
+<span class="like-friend-list"></span>
+</span>
+</div>
 </dd>
 </dl>
 <?php endforeach; ?>
 </div></div>
 <?php /* }}} */ ?>
 <?php endif; ?>
+
+<script id="LikelistTemplate" type="text/x-jquery-tmpl">
+<table>
+<tr style="padding: 2px;">
+<td style="width: 48px; padding: 2px;"><a href="${profile_url}"><img src="${profile_image}" width="48"></a></td>
+<td style="padding: 2px;"><a href="${profile_url}">${name}</a></td>
+</tr>
+</table>
+</script>
+
+<div id="likeModal" class="modal hide">
+  <div class="modal-header">
+    <h1>「いいね！」と言っている人</h1>
+  </div>
+  <div class="like-modal-body">
+  </div>
+  <div class="modal-footer">
+    <a href="#" class="btn close" data-dismiss="modal" aria-hidden="true">閉じる</a>
+  </div>
+</div>
